### ISSUE TITLE
Failed to connect prisma to existing database

### ISSUE THREAD

**Issue by tonyxiao** [prisma/prisma#2781]
**Describe the bug**

While running `prisma deploy` from a database generated via prisma introspection, I get.

```
Deploying service `default` to stage `default` to server `local` !

ERROR: Syntax error while parsing GraphQL query. Invalid input "uuid_generate_v4(", expected Value or Argument (line 2, column 35):
  id: ID! @unique @default(value: uuid_generate_v4())
```

**To Reproduce**
The following datamodel.graphql excerpt was generated from an existing postgres database. It is however not valid upon attempting to run `prisma deploy`
```
type Account @pgTable(name: "account") {
  id: ID! @unique @default(value: uuid_generate_v4())
```

**Expected behavior**

For deploy to succeed

**Screenshots**

**Versions (please complete the following information):**
 - OS: `OS X High Sierra`
 - `prisma` CLI: prisma/1.12.0 (darwin-x64) node-v10.6.0
 - Prisma Server: `1.12`

**Additional context**
My postgres database uses `uuid` as a primary key.

**Comment by wieseljonas** [comment]
I have the exact same issue

**Comment by marktani** [comment]
This was fixed in `1.14`.
